### ISSUE TITLE
Add a `run(*actors)` method to the SDK

### ISSUE THREAD

**Issue by leandro-lucarella-frequenz** [frequenz-floss/frequenz-sdk-python#36]
### What's needed?

Probably all applications will need to start some actors and then wait for them to finish indefinitely. So far this is on the SDK users, but it could be handled by SDK. Also this allows us to find better methods to wait for all actors (or tasks) to finish.

In this case the SDK can also provide sanity checks, like the `microgrid_api` is properly initialized, before starting all the actors.

### Proposed solution

A `run()` function would be (pseudo-code):

```py
async def run(*actors: Actor) -> None:
    # Check the microgrid_api is initialized (it will raise an exception if it isn't)
    microgrid_api.get_microgrid_api()
    await asyncio.gather(actor.join() for actor in actors)
```

Then clients will be like:

```python
from frequenz import sdk
from frequenz.sdk import microgrid_api

async def main() -> None:
    await microgrid_api.initialize("host", 8080)
    actor1 = Actor1()
    actor2 = Actor2()
    actor3 = Actor3()

    await sdk.run(actor1, actor2, actor3)

if __name__ == '__main__':
    asyncio.run(main())
```

### Use cases

All apps have to di the join() manually, it bloats the app code (and currently it has an extra issue because of #45).

### Alternatives and workarounds

Do the join() manually.

Further improvements to this:

* #196

**Comment by leandro-lucarella-frequenz** [comment]
After talking to @daniel-zullo-frequenz we decide to split this issue in 2 (new issue: #196). We'll work together in this but he'll focus on this and I'll focus on:

* #196 
* #45 

As they require more deep surgery and a better understanding on the actor framework.
